### PR TITLE
make greet test exercise

### PR DESCRIPTION
**Pull request by D3nnis249**
add greet test exercise

### CODE REVIEW

File: greet.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function greet(coach) {
+  if ("Martin" === coach || "Thomas" === coach || "Stefan" === coach) {
+    return "Hello Coach!";
+  } else {
+    return "Hello stranger!";
+  }
+}
+
+console.log(greet("Martin"));
```
Comment (mpagels) [inline greet.js]: You can and should remove this here ;-)

File: greet.spec.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { greet } from "./greet";
+
+describe("test for the greet function", () => {
+  it("if the name equal Thomas or Martin or Stefan return 'Hello Coach'", () => {
+    const result = greet("Thomas");
+    expect(result).toBe("Hello Coach!");
```
Comment (mpagels) [inline greet.spec.js:6]: Very good test.
To be confident the most, I recommend adding two more tests at least. To test if the result is the same when providing `Martin` and `Stefan`. Right now, you just test `Thomas`

File: greet.spec.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { greet } from "./greet";
+
+describe("test for the greet function", () => {
```
Comment (mpagels) [inline greet.spec.js:3]: Missing the test that tests when provider another name than the coaches?

**Review by mpagels — CHANGES_REQUESTED** [review]
Very good start here, Dennis. I like it very much. But missing some tests. Please check it out. I left some comments.

And very good that you did provide a `pull request description`. That helps a lot to know on what to focus here.

File: greet.js (hunk 1 above)
Comment (D3nnis249) [reply greet.js]: done

File: greet.spec.js (hunk 2 above)
Comment (D3nnis249) [reply greet.spec.js:6]: done

File: greet.spec.js (hunk 3 above)
Comment (D3nnis249) [reply greet.spec.js:3]: done

**Comment by D3nnis249** [comment]
Thank you for your review! :) I´ve added some tests for more coaches and strangers.

**Pull request merged**